dangerzone/isolation_provider/container.py

```diff
@@ -67,7 +67,10 @@ def get_runtime_version() -> Tuple[int, int]:
         cmd = [runtime, "version", "-f", query]
         try:
             version = subprocess.run(
-                cmd, capture_output=True, check=True
+                cmd,
+                startupinfo=get_subprocess_startupinfo(),
```

Comment:
I see that this is used in other parts of the code, so see the previous comment mainly for my understanding, it can come in a later comment.

Reply:
My understanding is that with this attribute, you can execute a Windows command without a controlling terminal. I haven't looked into details on how it works though.

This may be something we want to enforce for every command that we run in the future, so that we don't bump into this error again.